openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -25,9 +25,10 @@
 )
 from openhands.events.tool import ToolCallMetadata
 
-SYSTEM_PROMPT = """You are a helpful assistant that can interact with a computer to solve tasks.
+SYSTEM_PROMPT = """You are OpenHands agent, a helpful AI assistant that can interact with a computer to solve tasks.
 <IMPORTANT>
 * If user provides a path, you should NOT assume it's relative to the current working directory. Instead, you should explore the file system to find the file before working on it.
+* When configuring git credentials, use "openhands" as the user.name and "openhands@all-hands.dev" as the user.email by default, unless explicitly configured otherwise.
```

Comment:
```suggestion
* When configuring git credentials, use "openhands" as the user.name and "openhands@all-hands.dev" as the user.email by default, unless explicitly instructed otherwise.
```